src/create-render-context.js

```diff
@@ -172,6 +177,21 @@ const patchTimers = (): void => {
     patchCallbackFnWithGate(window, "requestAnimationFrame", "__SSR_ACTIVE__");
 };
 
+const createVirtualConsole = () => {
+    const virtualConsole = new VirtualConsole();
+    virtualConsole.on("jsdomError", (e: Error) => {
+        if (e.message.indexOf("Could not load img") >= 0) {
+            // We know that images cannot load. We're deliberately blocking
+            // them.
+            return;
+        }
+        // eslint-disable-next-line no-console
+        console.error(`JSDOM: ${e.stack}`);
```

Comment:
@jeresig we still log these errors to the console. However, we deliberately hide any complaining about images loading, because we already know these really aren't an issue in our world.

Reply:
Ah nice, thank you!